bindings/python/scripts/parse.py

```diff
@@ -121,23 +121,23 @@ def generate_parsed_info(node):
     depth = node["depth"]
 
     # @TODO: fix
-    if cursor.spelling != "":
-        parsed_info["depth"] = depth
-        parsed_info["line"] = cursor.location.line
-        parsed_info["column"] = cursor.location.column
-        parsed_info["kind"] = cursor.kind.name
-        parsed_info["name"] = cursor.spelling
-        if cursor.type.kind.spelling != "Invalid":
-            parsed_info["element_type"] = cursor.type.kind.spelling
-        if cursor.access_specifier.name != "INVALID":
-            parsed_info["access_specifier"] = cursor.access_specifier.name
-        if cursor.result_type.spelling != "":
-            parsed_info["result_type"] = cursor.result_type.spelling
-        if cursor.brief_comment:
-            parsed_info["brief_comment"] = cursor.brief_comment
-        if cursor.raw_comment:
-            parsed_info["raw_comment"] = cursor.raw_comment
-        parsed_info["members"] = []
+    # if cursor.spelling != "":
+    parsed_info["depth"] = depth
```

Comment:
Change base of PR. Why are these changes showing up in this PR? :confused:

Reply:
Oops, needed a rebase.